samples/graph-appcatalog-lifecycle/csharp/AppCatalogSample/Helper/AppCatalog.cs

```diff
@@ -0,0 +1,394 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using AdaptiveCards;
+
+namespace AppCatalogSample.Helper
+{
+  public class AppCatalog
+    {
+        public static  string gToken { get; set; }
+        public string GetToken(string token)
+        {
+            gToken = token;
+            return token;
+        }
+
+        public static async Task<IList<TeamsApp>> GetAllapp()
+        {
+            try
+                {
+
+                if (string.IsNullOrEmpty(gToken))
+                    return null;
+                GraphClient graphClient = new GraphClient(gToken);
+                var globalgraphClient = graphClient.GetAuthenticatedClient();
+                var teamsApps = await globalgraphClient.AppCatalogs.TeamsApps
+                   .Request()
+                   .Filter("distributionMethod eq 'organization'")
+                   .GetAsync();
+
+                return teamsApps.CurrentPage;
+                }
+            catch (Exception ex)
+                {
+                Console.WriteLine(ex.Message);
+
+                }
+
+        }
+
+        public static async Task<IList<TeamsApp>> AppCatalogById()
+        {
+            try
+                {
+                if (string.IsNullOrEmpty(gToken))
+                    return null;
+                var listApp = AppCatalog.GetAllapp().ConfigureAwait(false).GetAwaiter().GetResult();
+                var id = listApp.Where(x => x.DisplayName == "AppCatalog").Select(x => x.Id).FirstOrDefault();
+                GraphClient graphClient = new GraphClient(gToken);
```

Comment:
This is repeatative in all the methods. Could you please have single client which is used across?